src/Services/Passenger/src/Passenger/Passengers/Models/ValueObjects/GenericValueObject.cs

```diff
@@ -0,0 +1,13 @@
+namespace Passenger.Passengers.Models.ValueObjects;
```

Comment:
Please completely remove this generic base for this value object.

Reply:
done